assets/prometheus-k8s/rules.yaml

```diff
@@ -362,6 +362,14 @@ spec:
     rules:
     - expr: sum(openshift_build_total{job="kubernetes-apiservers",phase="Error"})/(sum(openshift_build_total{job="kubernetes-apiservers",phase=~"Failed|Complete|Error"}))
       record: build_error_rate
+  - name: openshift-sre.rules
+    rules:
+    - expr: sum(apiserver_request_count{job="apiserver"}) by (code)
+      record: code:apiserver_request_count:sum
+    - expr: kube_pod_status_ready{condition="true",namespace="kube-system",pod=~"etcd.*"}
+      record: kube_pod_status_ready:etcd
+    - expr: kube_pod_status_ready{condition="true",namespace="openshift-image-registry",pod=~"image-registry.*"}
```

Comment:
Same as for etcd pods.

Reply:
we decided we can grab current metrics to represent this data in telemeter so this will be removed same as the etcd pods metric